packages/block-library/src/gallery/transforms.js

```diff
@@ -177,56 +173,6 @@ const transforms = {
 		{
 			type: 'shortcode',
 			tag: 'gallery',
-
-			attributes: {
-				images: {
-					type: 'array',
-					shortcode: ( { named: { ids } } ) => {
-						if ( ! isGalleryV2Enabled() ) {
```

Comment:
There is no check in `transforms` whether the Gallery block is v2 or not. However, since the Gallery block v2 is enabled in all WordPress versions supported by the current Gutenberg plugin, I don't think it is necessary to add this check to `transforms`.